Makefile

```diff
@@ -138,6 +138,10 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
 
+.PHONY: test-unit
```

Comment:
Can you just reuse `test` above?

I think you are missing the dependency targets for `test-unit` so I think this may fail.

Reply:
I don't want it to generate, or format or vet...

Reply:
Ah, maybe I need the envtest dependency?